playground/backend/internal/code_processing/code_processing.go

```diff
@@ -114,13 +115,26 @@ func Process(ctx context.Context, cacheService cache.Cache, lc *fs_tool.LifeCycl
 	}
 }
 
+// getRunOrTestCmd return cmd instance based on the code type: unit test or example code
+func getRunOrTestCmd(valRes map[string]bool, executor *executors.Executor, ctxWithTimeout context.Context) *exec.Cmd {
```

Comment:
Usage of `Or` in the names of functions is not so good idea, let's rename it like `getExecuteCmd` and leave the same docstring – it explains the function quite good

Reply:
Done